src/screens/SUE/SueListScreen.tsx

```diff
@@ -174,11 +174,11 @@ export const SueListScreen = ({ navigation, route }: Props) => {
         }
         ListHeaderComponent={
           <>
-            <Wrapper>
+            <WrapperVertical>
               <Search setQueryVariables={setQueryVariables} placeholder={texts.filter.search} />
-            </Wrapper>
+            </WrapperVertical>
 
-            <Wrapper>
+            <WrapperVertical>
               <Filter
                 filterTypes={[
                   {
```

Comment:
There is an error in the dropdown here as well as on the event index screen. I think it is related to the dropdown component in general

https://github.com/user-attachments/assets/9956527c-e22b-45d9-8f59-8d8b038e193e

Reply:
fixed

<img width="383" alt="image" src="https://github.com/user-attachments/assets/b80266b0-d57e-43fe-8a0f-204c0fc0e7c3">